eng/pipelines/templates/stages/archetype-python-release.yml

```diff
@@ -46,6 +46,14 @@ stages:
                           PackageName: ${{artifact.name}}
                           ServiceName: ${{parameters.ServiceDirectory}}
                           ForRelease: true
+
```

Comment:
I could be wrong, but I don't think this will actually work without installing `tools/azure-sdk-tools` Have you tried an azure-template release using this?

Reply:
@scbedd sorry just seeing this comment. I think we discussed this offline. Did your debugging make any sense of it?

Reply:
Yeah...I have no explanation.

Reply:
[Yep. It's there](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3343636&view=logs&j=7bb69cc7-6f52-585e-8b13-5bae7c4a9fa4&t=7abbbe6b-d005-5c95-feb6-99ffe4e699e1&l=140)